.github/workflows/tests.yml

```
@@ -25,7 +25,7 @@ permissions:
 
 jobs:
   tap_syntax:
-    if: github.repository == 'Homebrew/homebrew-core'
+    if: github.repository == 'Homebrew/homebrew-core' && (github.event_name != 'pull_request' || !contains(github.event.pull_request.labels.*.name, 'CI-published-bottle-commits'))
```

I hear De Morgan talking to me.

```suggestion
    if: github.repository == 'Homebrew/homebrew-core' && !(github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'CI-published-bottle-commits'))
```